lib/embed.dart

```diff
@@ -781,6 +784,15 @@ class Embed {
     }
   }
 
+  void _virtualPageView(String id) {
```

Comment:
nit: Name this with a verb rather than a noun (`_sendVirtualPageView` rather than just `_virtualPageView`, for example).